main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       tree
+//  action 1:   the tree grows
+tree.grow();
+//  action 2:   the tree moves
+tree.move();
+//  action 3:   the tree creaks
+tree.creak();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       dog
+//  action 1:   the dog digs
+dog.dig();
+//  action 2:   the dog barks
+dog.bark();
+//  action 3:   the dog rollsOver
+dog.rollOver();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       phone
+//  action 1:   the phone rings
+phone.ring();
+//  action 2:   the phone vibrates
+phone.vibrate();
+//  action 3:   the phone notifiesUser
+phone.notifyUser();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       water
+//  action 1:   the water flows out 
+water.flowOut();
+//  action 2:   the water flows in
+water.flowIn();
+//  action 3:   the water trickles
+water.trickle();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       bird
+//  action 1:   the bird chirps
+bird.chirp();
+//  action 2:   the bird sings
+bird.sing();
+//  action 3:   the bird flys 
+bird.fly();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       dirtbike
+//  action 1:   the dirtbike runs
+dirtbike.run();
+//  action 2:   the dirtbike engages the brakes
+dirtbike.engageBrakes();
+//  action 3:   the dirtbike speeds up
+dirtbike.speedUp();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       chop saw
+//  action 1:   the chop saw spins the blade
+chopSaw.spinBlade();
+//  action 2:   the chop saw lowers
+chopSaw.lower();
+//  action 3:   the chop saw stops the blade
+chopSaw.stopBlade();
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       insect
+//  action 1:   the insect jumps
+insect.jumps();
+//  action 2:   the insect buzzes
+insect.buzz();
+//  action 3:   the insect flaps its wings 
+insect.flapWing();
```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**replace** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.